src/odemis/driver/test/xt_client_test.py

```diff
@@ -246,8 +245,8 @@ def setUpClass(cls):
                 cls.scanner = child
             elif child.name == CONFIG_FOCUS["name"]:
                 cls.efocus = child
-            elif child.name == CONFIG_STAGE["name"]:
-                cls.stage = child
+            # elif child.name == CONFIG_STAGE["name"]:
+            #     cls.stage = child
```

Comment:
I will remove these changes as soon as I have performed the last testing (after a first round of review)